src/Fhir.Anonymizer.AzureDataFactoryPipeline/src/FhirAzureConstants.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Fhir.Anonymizer.AzureDataFactoryPipeline.src
+{
+    public static class FhirAzureConstants
+    {
+        public const int KB = 1024;
+        public const int MB = KB * 1024;
+        public const int BlockBufferSize = 4 * MB;
+        public const int DefaultConcurrentCount = 3;
```

Comment:
Should the concurrent counts be same as parallelism of anonymization?

Reply:
Download or upload is in batch mode (4MB & 32MB), and also the download & upload is much faster(60MB/s for cross region) than processing(n * 100MB / min), looks like put 3 of them is enough? when executor is processing, there's at least 1~2 workers downloading and uploading.

Reply:
sounds good